### ISSUE TITLE
Feature Request: Port from tensorflow 1.15 to 2

### ISSUE THREAD

**Issue by TimiH** [davidADSP/SIMPLE#14]
Would be nice to just run the scripts themself in tensor flow 2. This way it would be a lot easier to avoid the docker hassle, since tf=1.15 is not available in python 3.9 anymore.

**Comment by vmunhoz** [comment]
yeah, I was trying to run it without docker and faced this as well. I'm really interested in this project, but I'm not familiar enough with all of this in order to try to make a pull request to update the libraries.

**Comment by pbtura** [comment]
I've been trying and the problem is harder than it looks. To get support for TF2 requires migrating to Stable-Baselines3. Significant portions of the code would have to be completely re-written. Not impossible for someone familiar with SB3, but well beyond what I could do.

**Comment by pbtura** [comment]
If anyone is interested, I think I finally made a breakthrough. The custom 'model' classes were a roadblock so I ended up bypassing them and just using the default stable_baselines3.common.policies.ActorCriticPolicy . I have the training working for sushigo and I plan to eventually update the other games as well. The code can be found here: https://github.com/pbtura/SIMPLE

Be aware that this is very much an experimental work in progress. I still have a TON of work to do updating the other games and testing to find out what else is broken.

**Comment by Fabbe** [comment]
@pbtura do you need help?
